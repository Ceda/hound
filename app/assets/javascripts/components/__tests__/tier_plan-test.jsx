import React from 'react';
import renderer from 'react-test-renderer';

import TierPlan from '../tier_plan.js';

it('renders appropriately (current plan)', () => {
  const plan = {
    name: "Chihuahua",
    price: 49,
    allowance: 4,
    current: true
  }

  const component = renderer.create(
    <TierPlan
      isCurrent={true}
      isNew={true}
      key={plan.name}
      plan={plan}
    />
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
