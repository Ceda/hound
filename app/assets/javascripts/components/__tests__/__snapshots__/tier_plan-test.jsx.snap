exports[`test renders appropriately (current plan) 1`] = `
<div
  className="plan plan-vertical current new">
  <div
    className="marker-wrapper">
    <span
      className="current-plan">
      Current Plan
    </span>
  </div>
  <div
    className="marker-wrapper">
    <span
      className="new-plan">
      New Plan
    </span>
  </div>
  <div
    className="plan-divider">
    <h5
      className="plan-title">
      Chihuahua
    </h5>
    <div
      className="plan-allowance">
      Up to 
      <strong>
        4
      </strong>
       Repos
    </div>
  </div>
  <div
    className="plan-price">
    $
    49
     
    <small>
      month
    </small>
  </div>
</div>
`;
